packages/common/container-definitions/package.json

```diff
@@ -59,7 +59,7 @@
 		"@fluidframework/core-interfaces": "workspace:~",
 		"@fluidframework/driver-definitions": "workspace:~",
 		"@fluidframework/protocol-definitions": "^3.2.0-237840",
-		"events": "^3.1.0"
+		"@types/events_pkg": "npm:@types/events@^3.0.0"
```

Comment:
I find it a bit funky to see @types/ packages as regular dependencies. I wouldn't have expected an `import type { ... } from ''events_pkg"` (which isn't listed anywhere in dependencies) to work.

Reply:
Yep. It is funky. We should stop using "events" package. I filed AB#7377 to replace with the tree version and this would be dealt with then.

Reply:
I also did confirm that type only imports only require type declarations. If "type" is removed, then "events_pkg" must be present.
tsc sees the ".js" import and looks for a .d.ts in expected place but doesn't care about actual ".js". It won't find a node_modules/events_pkg/index.d.ts and will check for node_modules/@types/events_pkg/index.d.ts.